docs/en-US/Get-PlasterTemplate.md

```diff
@@ -19,15 +19,18 @@ Get-PlasterTemplate [[-Path] <String>] [[-Name] <String>] [[-Tag] <String>] [-Re
 
 ### IncludedTemplates
 ```
-Get-PlasterTemplate [-IncludeInstalledModules] [[-Name] <String>] [[-Tag] <String>] [<CommonParameters>]
+Get-PlasterTemplate [-IncludeInstalledModules] [-ListAvailable] [[-Name] <String>] [[-Tag] <String>] [<CommonParameters>]
```

Comment:
Take a look at the Tag parameter section starting around line 169.  I think we need a section like that for this new parameter.  Or if you want, I can add that later.  Up to you.

Reply:
To be honest, I need to refresh my memory on how this doc works with platyPS.  I think platyPS generates this once (up front) but maybe it updates it with parameter info on every build?

Reply:
https://github.com/PowerShell/Plaster/blob/master/build.psake.ps1#L235-L317

It looks like all of the docs are generated at build time.